pkg/i18n/japanese.go

```diff
@@ -597,5 +597,6 @@ func japaneseTranslationSet() TranslationSet {
 			// CompletePrompt:              "Bisect complete! The following commit introduced the change:\n\n%s\n\nDo you want to reset 'git bisect' now?",
 			// CompletePromptIndeterminate: "Bisect complete! Some commits were skipped, so any of the following commits may have introduced the change:\n\n%s\n\nDo you want to reset 'git bisect' now?",
 		},
+		ConfirmPromptTitle: "確認する",
```

Comment:
```suggestion
		ConfirmPromptTitle: "確認",
```

Reply:
Thank you! Updated in a new commit: https://github.com/jesseduffield/lazygit/pull/2017/commits/ae51fc97be96f6eeb77de59844b9d876f5189a3e

Reply:
I ended up dropping this commit based on PR feedback: https://github.com/jesseduffield/lazygit/pull/2017#discussion_r912549938